database/migrations/2020_01_01_135000_create_business_category_place_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBusinessCategoryPlaceTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('business_category_place', function (Blueprint $table) {
+            $table->id();
+            $table->foreignIdFor(\Tipoff\Seo\Models\BusinessCategory::class);
+            $table->foreignIdFor(app('place'));
+            $table->timestamps();
+
+            $table->unique(['business_category','place']);
```

Comment:
See comment here:

- https://github.com/tipoff/seo/issues/23#issuecomment-799934762

There should be a polymorphic relationship between business_categories and place_details and gmb_details

Reply:
So this business_category_place table should not exist. A new table for the polymorphic relationships should be created. I'm not sure what to call it.

Reply:
what is the -able?  detailable or  categorizable?

Reply:
Hmm, let's use categorizable. I don't know what the name of that table should be. Maybe businesscategory_records like we did in the tipoff/statuses pacakge

Reply:
Why not just use the business_categories table and make it a morph for anything that needs a business category?  Would just have name, categorizable_id, categorizable_type.

Reply:
Because I want that to be unique. I want to know what Business Categories are morphed to each. Not have 500 business categories for the same thing.

Reply:
If you have a location with a morphed business category, that will be unique to that model: categorizable_id=<location_id> categorizable_type='/tipoff/Locations/Models/Location', name=<gmb_category>

Reply:
Correct. I don't want it to be unique to the model. I want business_cateogries to be unique names and another model that will be unique to the model.